src/index.html

```diff
@@ -7,8 +7,66 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    >
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap"
+      rel="stylesheet"
+    >
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#home">
+        <img
+          src="./images/logo.png"
+          alt="Moyo logo"
+          class="logo"
+      >
+      </a>
+      <nav class="nav">
+        <ul class="nav__links">
+          <li class="nav__list">
+            <a href="#" class="nav__link is-active">Apple</a>
+          </li>
+          <li class="nav__list">
+            <a href="#" class="nav__link">Samsung</a>
+          </li>
+          <li class="nav__list">
```

Comment:
```suggestion
      <a href="#home">
        <img
          src="./images/logo.png"
          alt="Moyo logo"
          class="logo"
      >
      </a>
      
      <nav class="nav">
        <ul class="nav__links">
          <li class="nav__list">
            <a href="#" class="nav__link is-active">
              Apple
             </a>
          </li>
          
          <li class="nav__list">
            <a href="#" class="nav__link">
              Samsung
            </a>
          </li>
          
          <li class="nav__list">
```
fix code style all over the page